crates/attestation/src/attestation.rs

```diff
@@ -93,10 +87,8 @@ pub enum VerificationError {
     },
     #[error("the mock attestation is invalid per definition")]
     InvalidMockAttestation,
-    #[error("the allowed mpc image hashes list is empty")]
-    EmptyAllowedMpcImageHashesList,
-    #[error("the allowed mpc laucher compose hashes list is empty")]
-    EmptyAllowedMpcLauncherComposeHashesList,
+    #[error("custom error: `{0}`")]
+    Custom(String),
```

Comment:
this is so that down-stream users can provide their own error messages. The first down-stream user is the mpc-crate, so errors that previously had names, now become this Custom variant